core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -125,10 +126,12 @@ private class ClientActor(driverArgs: ClientArguments, conf: SparkConf)
         }
         System.exit(0)
     }
+    // scalastyle:on println
   }
 
   override def receiveWithLogging: PartialFunction[Any, Unit] = {
 
+    // scalastyle:off println
     case SubmitDriverResponse(success, driverId, message) =>
       println(message)
```

Comment:
It seems like a number of cases like this might be bad uses of println? I don't immediately see why this isn't a normal log